### DISCUSSION TITLE
Request function (Save all settings to file)

### DISCUSSION THREAD

**Discussion by torerobo** [doublecmd/doublecmd#1139] [Suggestions]
**Request function (Save all settings to file)**

For a very long time I customized the look for myself. When removing or installing DC on a new OS, you have to reconfigure in a new way.
Please: add the function to save all settings to the file (conveniently when migrating)

**Comment by epexa** [comment]
@torerobo you can just copy the file:
/home/#YOUR_USERNAME/.config/doublecmd/doublecmd.xml
it stores your settings!

**Comment by savchenko** [comment]
What @epexa has said; just keep in mind that `doublecmd.xml` exposes sensitive paths, etc.

**Comment by torerobo** [comment]
> @torerobo you can just copy the file:
> /home/#YOUR_USERNAME/.config/doublecmd/doublecmd.xml
> it stores your settings!

Thank you for the decision!  Developers could make a similar "file-save settings" option in the DC GUI.  No one understands simple things.

**Comment by timur-g** [comment]
IMO this is WontFix. As it's clear that settings are in doublecmd.xml, once you learn it and you had better do it.
